app/views/users/backup_code_setup/create.html.erb

```diff
@@ -49,6 +47,14 @@
   </div>
 </div>
 
-<%= form_tag(backup_code_continue_path, method: :patch) do %>
-  <%= button_tag t('forms.buttons.continue'), type: 'submit', class: 'usa-button usa-button--big usa-button--wide' %>
-<% end %>
+<%= simple_form_for(:backup_code_accepted_form, url: backup_code_continue_path, method: :patch) do |f| %>
+  <%= render ValidatedFieldComponent.new(
+        form: f,
+        name: :backup_code_notice_accepted,
+        as: :boolean,
+        label: t('forms.backup_code.saved'),
+        label_html: { class: 'margin-y-2' },
```

Comment:
I'm curious about the margin on the label here, since I'd assume the standard component spacing would be fine. Was this in the design? I think I'd rather push back or at least double-check with UX that the non-standard spacing was intentional.

```suggestion
```

Reply:
Actually, I see that I might have jumped to conclusions here, overlooking the fact that this label is for the checkbox, so the spacing might be needed. I retract my previous suggestion. (I hadn't actually reviewed close enough to test the page locally)

Reply:
Re-adding.